wcomponents-core/src/main/java/com/github/bordertech/wcomponents/container/TransformXMLInterceptor.java

```diff
@@ -93,7 +93,7 @@
 	 */
 	@Override
 	public void preparePaint(final Request request) {
-		doTransform = isPerformTransform();
+		doTransform = true;

```

Comment:
Yeah, OK. It is unlikely that the header would change between requests within the same session.